common/artifacts/000_wc_artifacts.txt

```diff
@@ -792,14 +1083,101 @@ dark_crystal = {
 	stacking = no
 	indestructible = yes
 
-	learning = 3
-	combat_rating = 10
+	learning = 4
+	combat_rating = 15
 
-	monthly_character_prestige = 2
+	monthly_character_prestige = 1.5
 
 	text_effect_powerful_magical_artifact = 1
 
 	picture = "GFX_inv_misc_orb_03"
+	slot = relic
+}
+horn_of_cenarius = {
+	quality = 5
+	flags = { misc powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+	}
+	allowed_gift = {
+		always = yes
+	}
+
+	stacking = no
+	indestructible = yes
+
+	health = 0.5
+
+	monthly_character_prestige = 0.7
+	monthly_character_piety = 0.4
+
+	druidism_group_opinion = 10
+	dryad_group_opinion = 10
+	night_elf_group_opinion = 10
```

Comment:
...that's... OP... +20 to opinion max. And there's no religion limitations

Reply:
yeah, it gives opinion, but almost nothing else... should i reduce opinions by 5 and add, lets say, +1 diplo/+1 learning?

Reply:
reduced to +5 each, changed health from +0.5 to +0.75 and added druid-class limitation